include/pal_math.h

```diff
@@ -210,6 +210,48 @@ void p_mul_f32(const float *a, const float *b, float *c,
 void p_mac_f32(const float *a, const float *b, float *c,
                int n, int p, p_team_t team);
 
+/*logic and the two vectors: c =  a[n-1:0] & b[n-1:0]  */
+void p_and_f32(const float *a, const float *b, float *c,
```

Comment:
You're right. Brain fart. Fixed that, and pushed again. But I don't remember seeing a make error regarding this.